Automattic/sensei
# Change onboarding design

Fixes #5845
Based on https://github.com/Automattic/sensei/pull/5824

### Changes proposed in this Pull Request

* It updates the new onboarding design.

### Testing instructions


* Go to `/wp-admin/admin.php?page=sensei_setup_wizard`, and navigate through the whole onboarding.
* Make sure it works properly and with the new design.


### Screenshot / Video

@Luchadores, I think it would be nice to have a more vertical image for the usage tracking step, right? See the following video. ✅ https://github.com/Automattic/sensei/pull/5847/commits/2b2f52e7f4615ac15c4595a2e76604e6d8497a23

https://user-images.githubusercontent.com/876340/193918527-a478f915-5f86-43d4-ab37-8bb9cafd5b94.mov

https://user-images.githubusercontent.com/876340/193918423-e798f3e7-a532-4fb9-a14f-5b4daf8acec7.mov
